src/helpers/BapiClient.ts

```diff
@@ -698,6 +703,24 @@ export class BapiClient {
       ),
   };
 
+  public readonly redirects = {
+    get: (params: GetRedirectsEndpointParameters = {}) =>
+      this.execute(createGetRedirectsEndpointRequest(params)),
+
+    post: (url: string) => {
```

Comment:
Should we maybe name this better than `POST`, what about `match`?

Reply:
Yup, sounds good